apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30709.132
+# Visual Studio Version 17
```

Comment:
Is it OK that we do this now? Everyone needs to update VS.

@jskeet what do you think?